eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py

```diff
@@ -64,14 +58,18 @@ def get_all_campaigns():
   def get_one_campaign(id):
     return Campaign.query.get(id)
 
-  def __repr__(self):
+  @staticmethod 
+  def get_campaign_by_name(name):
+    return Campaign.query.filter_by(name=name).first()
+
+  def __repr(self):
     return '<id {}>'.format(self.id)
```

Comment:
This is where we define how we print objects of this class. In this case, if we print an instance of the campaign class, it will print in the format <id 5> (replace 5 with whatever Id is associated with this particular instance.